glusterd2/commands/volumes/volume-expand-txn.go

```diff
@@ -207,13 +207,29 @@ func updateVolinfoOnExpand(c transaction.TxnCtx) error {
 	} else {
 		// Create new Sub volumes with given bricks
 		subvolIdx := len(volinfo.Subvols)
-		for i := 0; i < len(newBricks)/newReplicaCount; i++ {
-			idx := i * newReplicaCount
+		bricksCount := newReplicaCount + volinfo.Subvols[0].ArbiterCount
+		for i := 0; i < len(newBricks)/bricksCount; i++ {
```

Comment:
can we have `len(newBricks)` calculation outside to avoid computing of this value each time.

Reply:
done